src/main/java/com/synopsys/integration/jenkins/scan/service/scan/ScanStrategyFactory.java

```diff
@@ -0,0 +1,52 @@
+package com.synopsys.integration.jenkins.scan.service.scan;
+
+import com.synopsys.integration.jenkins.scan.global.ApplicationConstants;
+import com.synopsys.integration.jenkins.scan.global.Utility;
+import com.synopsys.integration.jenkins.scan.global.enums.ScanType;
+import com.synopsys.integration.jenkins.scan.service.scan.blackduck.BlackDuckParametersService;
+import com.synopsys.integration.jenkins.scan.service.scan.coverity.CoverityParametersService;
+import com.synopsys.integration.jenkins.scan.service.scan.polaris.PolarisParametersService;
+import hudson.model.TaskListener;
+import java.util.Map;
+
+public class ScanStrategyFactory {
```

Comment:
This class **shouldn't** be inside `service.scan `package. Rather move it to another package named `strategy` which would be the same directory level of `service`.

Reply:
Packaged and updated accordingly